### DISCUSSION TITLE
Is it possible to not reload map when new data arrives?

### DISCUSSION THREAD

**Discussion by DanielBustillos** [thingsboard/thingsboard#10478] [Q&A]
I'm encountering an _issue_ with the OpenStreetMap widget in [specific platform/tool you are using]. Whenever new data arrives and the map updates, I notice two things:

- The map briefly blinks or flashes white.
- It automatically resets to the original zoom level instead of maintaining the current zoom state.

This behavior occurs consistently at certain zoom levels, disrupting the user experience as it requires re-adjusting the zoom after every data update.

**Steps to Reproduce:**

1. Zoom into a specific level on the OpenStreetMap widget.
2. Wait for new data to arrive or trigger a data update manually.
3. Observe that the map blinks white and then resets to the initial zoom level.

**Questions:**

Can this behavior be configured or prevented to maintain the zoom level after new data is received?

**Desired Outcome:**
I would like the map to maintain its current zoom level and not blink white upon receiving new data, ensuring a seamless experience for users monitoring real-time data.

Any suggestions, workarounds, or fixes for this issue would be greatly appreciated. Thank you!

**Comment by maaaaaaaaaaaaaan** [comment]
Do you have "Fit map bounds to cover all markers" enabled? Try to disable that, but then the start position of the map is fixed to "default map zoom level" and "default map center position".

**Comment by noonhub** [comment]
Bumping this, I have tried adjusting settings as well as using a different map provider and nothing seems to fix this
